Makefile

```diff
@@ -51,5 +51,4 @@ sonar:
 
 .PHONY: sonar-pr-analysis
 sonar-pr-analysis:
-	mvn verify -Dskip.unit.tests=true -Dskip.integration.tests=true
-	#mvn sonar:sonar -P sonar-pr-analysis #temporary until sonar available for Java 21
\ No newline at end of file
+	mvn verify -Dskip.unit.tests=true -Dskip.integration.tests=true
```

Comment:
If you left a blank line after this, it would not show this line to have changed.